overlord/snapstate/handlers_prereq_test.go

```diff
@@ -680,7 +680,7 @@ func (s *prereqSuite) TestDoPrereqBaseNoRevision(c *C) {
 	s.state.Lock()
 	defer s.state.Unlock()
 	c.Check(chg.Status(), Equals, state.ErrorStatus)
-	c.Check(chg.Err(), ErrorMatches, `cannot perform the following tasks:\n.*- test \(cannot install snap base "some-base": no snap revision available as specified\)`)
+	c.Check(chg.Err(), ErrorMatches, `cannot perform the following tasks:\n.*- test \(cannot install snap base "some-base": cannot install snap "some-base": no snap revision available as specified\)`)
```

Comment:
this change to errors is weird with the repetition, it would be good to work to avoid it somehow